public/app/themes/clarity/inc/amazon-s3-and-cloudfront-signing.php

```diff
@@ -129,11 +143,11 @@ public function getCloudfrontPublicKeyId(): string
         $public_key_short = substr($public_key_sha256, 0, 8);
 
         // Find the matching array entry for the public key.
-        $public_key_id_and_comment = array_filter($cloudfront_public_key_object, fn ($key) =>  $key['comment'] === $public_key_short && !empty($key['id']));
+        $public_key_id_and_comment = array_filter($cloudfront_public_key_object, fn ($key) => $key['key'] === $public_key_short && !empty($key['id']));
```

Comment:
@EmilyHazlehurst I think this needs to be reverted. Maybe causing these errors https://ministryofjustice.sentry.io/issues/5254928665/?project=4506870789505024&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=1